From: Torben Hohn <torben.hohn@linutronix.de>
Date: Mon, 17 Dec 2018 08:55:31 +0100
Subject: dozer is a memory debugger. see: https://github.com/mgedmin/dozer

Only useful in --devel mode. And its required, that some dependencies
are installed manually into the initvm

--------------------------------------------------------------------
apt install python3-pip
pip3 install Dozer
/etc/init.d/elbe-daemon restart
--------------------------------------------------------------------

then go to: http://localhost:7587/soap/_dozer/index

Signed-off-by: Torben Hohn <torben.hohn@linutronix.de>
---
 elbepack/daemons/soap/__init__.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/elbepack/daemons/soap/__init__.py b/elbepack/daemons/soap/__init__.py
index 4ecf6cde9..a0cbaf1e5 100644
--- a/elbepack/daemons/soap/__init__.py
+++ b/elbepack/daemons/soap/__init__.py
@@ -10,6 +10,7 @@ import sys
 
 from beaker.middleware import SessionMiddleware
 from cherrypy.process.plugins import SimplePlugin
+from dozer import Dozer
 
 from elbepack.projectmanager import ProjectManager
 
@@ -53,5 +54,5 @@ def get_app(engine):
                    in_protocol=Soap11(validator='lxml'),
                    out_protocol=Soap11())
 
-    wsgi = WsgiApplication(app)
+    wsgi = Dozer(WsgiApplication(app))
     return MySession(wsgi, app.pm, engine)
